module/roll/roll.js

```diff
@@ -390,17 +355,76 @@ export class DGPercentileRoll extends DGRoll {
       // use calculated target proficiency (effects and etc like aim + 20%)
       target = actorData.skills[this.key].targetProficiency || actorData.skills[this.key].proficiency;
       localizedKey = game.i18n.localize(`DG.Skills.${this.key}`);
+      skillPath = `system.skills.${this.key}`;
     }
     if (typedSkillKeys.includes(this.key)) {
       const skill = actorData.typedSkills[this.key];
       target = skill.proficiency;
       localizedKey = `${skill.group} (${skill.label})`;
+      skillPath = `system.typedSkills.${this.key}`;
     }
     if (this.key === "ritual") {
       target = actorData.sanity.ritual;
       localizedKey = game.i18n.localize(`DG.Skills.ritual`);
     }
-    return { target, localizedKey };
+    return { target, localizedKey, skillPath };
+  }
+
+  /**
+   * Create label based on result of roll
+   *
+   * todo: do we want make isInhuman more similar to base label?
+   *
+   * @returns {string}
+   */
+  createLabel() {
+    const startOfLabel = game.i18n.localize("DG.Roll.Rolling")
+      + ` <b>${this.localizedKey}`
+    const endOfLabel =`${game.i18n.localize("DG.Roll.Target")} ${this.effectiveTarget}`
+
+    const label = this.isInhuman
```

Comment:
```suggestion
    let label = this.isInhuman
```

You're gonna hate me but this was a mistake. This one *should* be a `let` because it is reassigned. It's a good reminder to not just blindly accept code suggestions too :)